code/game/world.dm

```diff
@@ -255,12 +255,25 @@ var/world_topic_spam_protect_time = world.timeofday
 	var/s = ""
 
 	if (config && config.server_name)
-		s += "<a href=\"[config.forumurl]\"><b>[config.server_name] &#8212; [MAIN_SHIP_NAME]</b>"
-		s += "<br><img src=\"[config.forumurl]/byond_hub_logo.jpg\"></a>"
-		// s += "<a href=\"http://goo.gl/04C5lP\">Wiki</a>|<a href=\"http://goo.gl/hMmIKu\">Rules</a>"
+		if(config.chaturl)
+			s += "<a href=\"[config.chaturl]\"><b>[config.server_name] &#8212; [MAIN_SHIP_NAME]</a></b>"
+		else
+			s += "<b>[config.server_name] &#8212; [MAIN_SHIP_NAME]</b>"
 		if(ticker)
 			if(master_mode)
-				s += "<br>Map: <b>[map_tag]</b>"
+				switch(map_tag)
+					if("Ice Colony")
```

Comment:
You dont need to do this here but we really need to move this sort of thing into map configs